spec/shell-session-spec.coffee

```diff
@@ -1,12 +1,14 @@
 describe "Shell session grammar", ->
   grammar = null
+  grammar_session = null
 
   beforeEach ->
     waitsForPromise ->
       atom.packages.activatePackage("language-shellscript")
 
     runs ->
-      grammar = atom.grammars.grammarForScopeName("text.shell-session")
+      grammar_session = atom.grammars.grammarForScopeName("text.shell-session")
```

Comment:
Pretty minor, but I don't think we use `_`s in identifier names anywhere else - it's usually all camelcased